types/param/genesis_param.go

```diff
@@ -0,0 +1,45 @@
+package param
+
+import (
+	"time"
+
+	"github.com/pactus-project/pactus/types/amount"
+)
+
+type GenParams struct {
+	BlockVersion              uint8         `cbor:"1,keyasint"  json:"block_version"`
+	BlockIntervalInSecond     int           `cbor:"2,keyasint"  json:"block_interval_in_second"`
+	CommitteeSize             int           `cbor:"3,keyasint"  json:"committee_size"`
+	BlockReward               amount.Amount `cbor:"4,keyasint"  json:"block_reward"`
+	TransactionToLiveInterval uint32        `cbor:"5,keyasint"  json:"transaction_to_live_interval"`
+	BondInterval              uint32        `cbor:"6,keyasint"  json:"bond_interval"`
+	UnbondInterval            uint32        `cbor:"7,keyasint"  json:"unbond_interval"`
+	SortitionInterval         uint32        `cbor:"8,keyasint"  json:"sortition_interval"`
+	FeeFraction               float64       `cbor:"9,keyasint"  json:"fee_fraction"` // Deprecated: Replaced by fix fee
```

Comment:
```suggestion
	FeeFraction               float64       `cbor:"9,keyasint"  json:"_"` // Deprecated: Replaced by fix fee
```

Reply:
Ignoring deprecated fields results in changing the genesis hash. should I do that?